cypress/support/util.ts

```diff
@@ -27,3 +27,19 @@ export function gqlOptions(body: any, additionalOptions?: any) {
     ...additionalOptions,
   };
 }
+
+export function expectHrefIdToBeInArray(link, ids: number[]) {
+  cy.wrap(link)
+    .invoke('attr', 'href')
+    .then((href) => {
+      const id = parseInt(href.match(/\d+/)[0]);
+      expect(ids).includes(id);
+    });
+}
```

Comment:
With something like this, we can keep things nice and readable by defining helper functions, but leaving the final `expect` inside the test itself.

So, for this we could do:
```suggestion
export function getFirstPathParam(link: JQuery<HTMLElement>) {
  return parseInt(link.attr('href').match(/\d+/)[0]);
}
```

and I'll add a comment below with an example usage.